docs/site/Deploying-with-pm2-and-nginx.md

```diff
@@ -98,16 +98,15 @@ section of documentation for detailed instructions.
 
 ## Deployment
 
-1. Register and start your application with pm2 using following command at the
+1. Register and start your application with `pm2` using following command at the
```

Comment:
nitpick: `using following` -> `using the following`

Reply:
Done